### ISSUE TITLE
Class/Property names look strange without an rdfs:label

### ISSUE THREAD

**Issue by situx** [dgarijo/Widoco#420]
We experience that without defining an rdfs:label for some of our classes that the output in the documentation of Widoco is somehow splitted (see the attached screenshot)
![image](https://user-images.githubusercontent.com/5874375/89890696-116b0f00-dbd4-11ea-9016-7d1ccff49563.png)
From what we can tell this happens when:
1. No label is defined
2. The classname begins with one or more capital letters possibly followed by an underscore.
Every classname you see in the screenshot will look similarly as AAA6 without labels defined.
Is this an intended behavior or is Widoco expecting defined labels in this case?

**Comment by dgarijo** [comment]
Yes, I have noticed this (issues #126 and #124). This is legacy from LODE, and requires a  little  digging to figure out.

Adding rdfs:labels with the human-readable serialization for the class and properties is preferred
